deploy/obs-csi-plugin/kubernetes/rbac-csi-obs-controller.yaml

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
All the CSI plugins should use the same account and roles, please modify later.

Reply:
it will be modified in another PR